.yarnrc

```diff
@@ -1,5 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
-# yarn lockfile v1
-
-
-yarn-path ".yarn/releases/yarn-1.22.22.cjs"
```

Comment:
Not sure what this file is doing but it breaks my build. If we remove it does it break yours @ImMPrada?

Reply:
No, I already tested it by making my build, and I agree, we should remove it!